pkg/workloads/manager.go

```diff
@@ -401,154 +423,8 @@ func (d *defaultManager) GetLogs(ctx context.Context, containerName string, foll
 	return logs, nil
 }
 
-func (d *defaultManager) findContainerByName(ctx context.Context, name string) (*rt.ContainerInfo, error) {
```

Comment:
There's some changes in this file which are the result of moving some private methods to the bottom to keep the public methods grouped together.